### PR TITLE
OCPBUGS-10270: Add a note that you have to include the default channel

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OCPBUGS-10270

Link to docs preview:
https://57487--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sherine-k Can you please review and let me know if this works for the note about the default channel being required?

Preview: https://57487--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

**Review by sherine-k — CHANGES_REQUESTED** [review]
Hi @bergerhoffer 
Sorry for the delay. I realise I never clicked the submit button!
Luckily github saved my comment!

File: modules/oc-mirror-creating-image-set-config.adoc [hunk 1]
```diff
@@ -63,8 +63,9 @@ mirror:
 <4> Set the channel to retrieve the {product-title} images from.
 <5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
 <6> Set the Operator catalog to retrieve the {product-title} images from.
-<7> Specify only certain Operator packages and channels to include in the image set. Remove this field to retrieve all packages in the catalog.
-<8> Specify any additional images to include in image set.
+<7> Specify only certain Operator packages to include in the image set. Remove this field to retrieve all packages in the catalog.
+<8> Specify only certain channels of the Operator packages to include in the image set. You must always include the default channel for the Operator package.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/oc-mirror-creating-image-set-config.adoc]: @sherine-k thanks! Updated, PTAL

https://57487--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-creating-image-set-config_installing-mirroring-disconnected

**Review by sherine-k — APPROVED** [review]
Yes! Thanks @bergerhoffer

**Pull request merged**